recipes/rnaprot/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "RNAProt" %}
+{% set version = "0.3" %}
+{% set sha256 = "18ffa9d569c63e3fc5d7a31427c7569d6021c67587a6838aea4ff3af7f0780e5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/BackofenLab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: "{{ sha256 }}"
+
+build:
+  number: 0
+  noarch: python
+  script: $PYTHON -m pip install . --ignore-installed --no-deps -vv
+
+requirements:
+  host:
```

Comment:
@michauhl is it possible to remove the version numbers for the requirements?

Reply:
For some packages I really need the specific version. For the rest I haven't tested. Is this a problem? In the other recipe there was a separate file with the version number: https://github.com/conda-forge/openmm-torch-feedstock/blob/beafcb48a45c632ec9cd79d299a06b300b117d7d/recipe/conda_build_config.yaml

Reply:
Ups somehow I "started a review". For some packages I really need the specific version. For the rest I haven't tested. Is this a problem? In the other recipe there was a separate file with the version number: https://github.com/conda-forge/openmm-torch-feedstock/blob/beafcb48a45c632ec9cd79d299a06b300b117d7d/recipe/conda_build_config.yaml

Reply:
If you need them it's okay but if possible they should be removed

Reply:
But isn't it better to specify exactly what you used for testing ? I often had this problem when installing just package names for some tools, but since the tool code was old it didn't work anymore with the newer package versions.

Reply:
It makes dependency resolution really tough (already a massive problem for conda) if you pin this tightly.

If you need specific versions for some packages it's fine I think but I would be really surprised if e.g. your code works only with python 3.8.

Reply:
also if you are worried about future versions can't you at least do e.g. `viennarna <=2.4.17`?

Reply:
True Python is not needed. But e.g. ViennaRNA, there are bug fixes from the last release which are important. Does >= or <= some version mean that it first tries to install the defined version ? If so we could probably use >= for most yes.

Reply:
It would try and install the highest version available if possible.

Reply:
Okay I will put a >= sign where I think it should not cause trouble for some time ...

Reply:
You think it's okay now ?

Reply:
The pinning needs to be way looser. For individual cases (i.e. if you rely on a particular feature introduced in e.g. `viennarna >= 2.4.17`) it's okay, but please remove the pin for everything else.

Reply:
Okay I will only keep the ones where I'm certain that newer versions don't work ...